projects/augeas/augeas_escape_name_fuzzer.cc

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Any chance to have this fuzz target in the upstream repository?

Reply:
I'd like to start the fuzz target here first and later I will add it in upstream repository.

Reply:
ok!